content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
 | [403][] | [JSON API error object][] | Forbidden - public provider curation disabled               |
 | [404][] | [JSON API error object][] | Provider not found or user not authorized to perform action |
 
+
+### Sample Request 
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/public/hashicorp/aws/versions/3.1.1
+```
+
+
+## Upload a Provider Platform (private provider)
```

Comment:
I feel like all of these ones that say upload should be "Add" or "Create". How do we do this for other API docs?